CHANGES.rst

```diff
@@ -4,20 +4,23 @@ Change History
 2.11.6 (unreleased)
 ===================
 
-- Nothing changed yet.
+- Add a new buildout option ``allow-unknown-extras`` to enable
+  installing requirements that specify extras that do not exist. This
+  needs a corresponding update to zc.recipe.egg. See `issue 457
```

Comment:
I've missed what needs updating in zc.recipe.egg.
If that's the case, there should be a change in the requirements in buildout's setup.py (after the zc.recipe.egg release).

Reply:
Yikes! Those changes didn't make it into the commit! They wound up in the installed copy.

I didn't add any tests specifically for zc.recipe.egg because it was just trivial copying arguments around.

Reply:
As long as the original tests didn't break... :-)

Note: if you needed to add an argument in zc.recipe.egg, I really want @jimfulton to have a quick look, as zc.recipe.egg rarely need changing.

Reply:
> As long as the original tests didn't break... :-)

They didn't, but the fact that I missed committing that file and everything still passed proves to me that it needed an integration test too. Added. :)